packages/atomic/cypress/integration/sort-dropdown.cypress.ts

```diff
@@ -0,0 +1,90 @@
+import {getApiRequestBody, getAnalytics} from '../utils/network';
+import {setUpPage} from '../utils/setupComponent';
+
+const sortDropdown = 'atomic-sort-dropdown';
+const criteria = ['relevancy', 'date descending'];
+
+describe('Sort Dropdown Component', () => {
+  function setup(attributes = '') {
+    setUpPage(`
+      <atomic-sort-dropdown ${attributes}>
+        <atomic-sort-criteria caption="relevance" criteria="${criteria[0]}"></atomic-sort-criteria>
+        <atomic-sort-criteria caption="mostRecent" criteria="${criteria[1]}"></atomic-sort-criteria>
+      </atomic-sort-dropdown>
+    `);
+    cy.wait(500);
+  }
+
+  function selectOption(value: string) {
+    return cy.get(sortDropdown).shadow().find('select').select(value);
+  }
+
+  function shouldRenderErrorComponent() {
+    cy.get(sortDropdown)
+      .shadow()
+      .find('atomic-component-error')
+      .should('exist');
+  }
+
+  it('should load', () => {
+    setup();
+    cy.get(sortDropdown).should('be.visible');
+  });
+
+  it('passes automated accessibility', () => {
+    setup();
+    cy.checkA11y(sortDropdown);
+  });
+
+  it('should display a label by default', () => {
+    setup();
+    cy.get(sortDropdown).shadow().find('label').should('exist');
+  });
+
+  it('should not display a label when display-label="false"', () => {
+    setup('display-label="false"');
```

Comment:
Is a `display-label` option necessary? Could we display a label of a `label` is specified, and not display one if it is an empty string?